RecodoDesktop/RecodoDesktop/VideoRecordingForm.xaml.cs

```diff
@@ -39,6 +32,15 @@ public VideoRecordingForm(RecorderService recorderService)
             _recorderService.Configure(_options);
         }
 
+        public VideoRecordingForm()
```

Comment:
Why are you duplicating the constructor?
DI should work ok in this case

Reply:
Just drop it for now

Reply:
I don't know why but it won't start method App_Startup when Startup="App_Startup"